README.md

```diff
@@ -48,11 +48,11 @@ As such, we encourage other scientific tool builders in academia or industry to
 this project. All code is freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT).
 
 
-Before extending cellxgene, we encourage you to reach out to us with ideas or questions. It might be possible that an 
-extension could be added directly to the explorer repo, which would make it available for a wider audience, or that it's on our 
+Before extending CZ CELLxGENE, we encourage you to reach out to us with ideas or questions. It might be possible that an 
```

Comment:
```suggestion
Before extending CZ CELLxGENE Explorer, we encourage you to reach out to us with ideas or questions. It might be possible that an 
```